src/get-started/java-binding.md

```diff
@@ -472,6 +481,66 @@ client.
 An example of API service tests can be found in
 [`ApiControllerTest`][apicontrollertest].
 
+## Using Libraries
+
+An Exonum service can use any third-party library as its dependency.
+
+The libraries, classes from which are used in Exonum public APIs,
+are included in the framework distribution, and can also be used by the service.
+Such libraries *must not* be packaged in its artifact.
+Use `provided` scope in the dependency declaration to achieve that in Maven.
+
+<!-- Otherwise multiple incompatible versions of the same class
+will be loaded by the plugin classloader and the application classloader, if they
+happen to need the same class -->
+
+The following Exonum dependencies *must* be used in the `provided` scope.
+They will not be changed in an incompatible way in a compatible Exonum release.
+Note that they do not have to be declared explicitly for your service already
+depends on "exonum-java-binding-core" which has them as transitive
+dependencies:
+
+- Exonum (exonum-java-binding-core, exonum-java-binding-common)
+- [Guice][Guice]
+- [Gson][gson]
+- [Vertx][vertx-web-docs] (vertx-web)
+- [Protobuf Java](https://github.com/protocolbuffers/protobuf/tree/master/java)
+- [Log4j 2][log4j2]
+- [PF4J](https://pf4j.org).
```

Comment:
Redundant dot in the end

Reply:
It ends a list of items.

Reply:
… but it isn't needed: https://en.oxforddictionaries.com/punctuation/bullet-points